examples/shopify/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "typescript.tsdk": "node_modules/typescript/lib"
+}
```

Comment:
I think this might be a bit too much for an example and that we should remove it. The GraphQL extension recommendation is okay because it's just a recommendation and it will enhance the example by making the GraphQL queries with comments get syntax highlighting